pkg/internal/pop/cache/cache.go

```diff
@@ -0,0 +1,69 @@
+package cache
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/AzureAD/microsoft-authentication-extensions-for-go/cache/accessor"
+	"github.com/AzureAD/microsoft-authentication-library-for-go/apps/cache"
+)
+
+const popTokenCacheFileName = "pop_cache.json"
+
+// getPoPCacheFilePath returns the file path for the PoP token cache.
+// This is separate from the authentication record cache file.
```

Comment:
Correct me if I'm wrong but this cache logic is not specific to just pop token right ? 
This is more to comply to msal ? If that the case, we should remove all pop reference in the name and description IMO.

Reply:
this is based on discussion with @weinong on a teams thread to apply these changes to pop flows only as non-PoP flows use `azidentity` & leverage it's cache implementation